templates/tarsnap-archive.erb

```diff
@@ -3,4 +3,4 @@
 title="$1"
 shift
 
-<%= @path %> --quiet -c -f $title.$(date +%Y%m%d%H%M) $@
+<%= scope.lookupvar("tarsnap::path") %> --quiet -c -f $title.$(date +%Y%m%d%H%M) $@
```

Comment:
you could just explicitly `include ::tarsnap` in `tarsnap::install`, that would pull path explicitly into the scope, making this unecessary.